docs/UserGuide.md

```diff
@@ -130,7 +130,7 @@ Edits an existing contact in the application.
 Format: `editPerson INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [g/GENDER] [d/DOB]`
 
 - Edits the person at the specified `INDEX`. The index refers to the index number shown in the displayed person list.
-  The index must be **a positive integer** 1, 2, 3, …​, and it must be within the range of the contact list index.
+  The index must be **a positive integer** 1, 2, 3, …​, and it must be within the range of the contact list index. This command is invalid if `INDEX` is a non-positive integer.
```

Comment:
It might be better to change index to code format `INDEX`, since the index in other commands uses code format. You can ignore this comment if it's not so necessary.
```suggestion
  The `INDEX` must be **a positive integer** 1, 2, 3, …​, and it must be within the range of the contact list index. This command is invalid if `INDEX` is a non-positive integer.
```